### ISSUE TITLE
warning : Accessing createClass via the main React package is deprecated

### ISSUE THREAD

**Issue by RamiroIsBack** [paypal/paypal-checkout#617]
while following the example provided for the usage in react, I get this warning in the console:

> lowPriorityWarning.js:38 Warning: Accessing createClass via the main React package is deprecated, and will be removed in React v16.0. Use a plain JavaScript class instead. If you're not yet ready to migrate, create-react-class v15.* is available on npm as a temporary, drop-in replacement. For more info see https://fb.me/react-create-class

> 
> printWarning | @ | lowPriorityWarning.js:38
> -- | -- | --
>   | lowPriorityWarning | @ | lowPriorityWarning.js:57
>   | get | @ | React.js:106
>   | register | @ | checkout.lib.js:7165
>   | Component.driver | @ | checkout.lib.js:5247
>   | ./src/components/presentational/PayPalButton.js | @ | PayPalButton.js:6
>   | __webpack_require__ | @ | bootstrap 9cb2f64…:669
>   | fn
> 

I guess is commig fron the way of creating the Button but I don't know how to do it with an import, or some other way,
here is my code:
thankyou!!
```
import React from 'react'
import ReactDOM from 'react-dom'
import paypal from 'paypal-checkout'
import PropTypes from 'prop-types'

**const Button = paypal.Button.driver('react', { React, ReactDOM })**

export default class PayPalButton extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      env: this.props.env,

      client: {
        sandbox: this.props.sandboxID,
        production: this.props.productionID
      },
      amount: this.props.amount,
      currency: this.props.currency,
      commit: this.props.commit
    }
  }

  payment(data, actions) {
    return actions.payment.create({
      transactions: [
        {
          amount: { total: this.state.amount, currency: this.state.currency }
        }
      ]
    })
  }

  onAuthorize(data, actions) {
    return actions.payment.execute().then((payment_data)=>{
      console.log('pagoHecho')
      var payment = {}
      payment.paid = true
      payment.cancelled = false
      payment.payerID = data.payerID
      payment.paymentID = data.paymentID
      payment.paymentToken = data.paymentToken
      payment.returnUrl = data.returnUrl
      // getting buyer's shipping address and email
      payment.address = payment_data.payer.payer_info.shipping_address
      payment.email = payment_data.payer.payer_info.email
      this.props.pagoHecho(payment)

    })
      .catch(err => {
        console.log ('error en PayPal'+err)
        throw err
      })
  }

  render() {
    return (
      <Button
        commit={ this.state.commit }
        locale= 'es_ES' //espaniolo
        style={{
          layout: 'vertical',  // horizontal | vertical
          size:   'responsive',    // medium | large | responsive
          shape:  'pill',      // pill | rect
          color:  'black'       // gold | blue | silver | black
        }}
        env={ this.state.env }
        client={ this.state.client }

        payment={ (data, actions) => this.payment(data, actions) }
        onAuthorize={ (data, actions) => this.onAuthorize(data, actions) }
        onCancel={this.props.onCancel}
        onError = {this.props.onError}
      />
    )
  }
}

PayPalButton.propTypes = {
  env: PropTypes.string.isRequired,
  sandboxID: PropTypes.string,
  productionID: PropTypes.string,
  amount: PropTypes.number.isRequired,
  currency: PropTypes.string.isRequired,
  commit: PropTypes.bool.isRequired
}
```

**Comment by bluepnume** [comment]
Thanks for the report, we'll take a look.
